src/app/modules/customer-management/services/customer.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from './../../../../environments/environment';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CustomerService {
+  baseUrl = `${environment.timeTrackerApiUrl}/customers`;
+
+  constructor(private http: HttpClient) {}
+
+  createCustomer(customerData): Observable<any> {
+    const body = {
+      ...customerData,
+      tenant_id: '4225ab1e-1033-4a5f-8650-0dd4950f38c8',
```

Comment:
@enriquezrene  Do we still need to set a tenant_id fake?

Reply:
I think @daros10 needs to talk to @macrisguncay to understand how to get the tenant_id from the session_storage. Good catch @jorgecod, ty.

Reply:
changes was applied except this.

Reply:
@daros10 you can get the real tenant id and bearer token from session storage, I implemented these methods in AzureAdB2CService, please check it and use them please.

Reply:
thank you @macrisguncay , @enriquezrene all review requesting was applied.